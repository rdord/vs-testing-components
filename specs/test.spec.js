import TestComponent from '@/test';
import { mount, shallowMount } from '@vue/test-utils';
import List from '@/list';

test('first spec', () => {
  const wrapper = mount(TestComponent, {
    propsData: { value: 'my value' }
  });
  expect(wrapper.html()).toMatchSnapshot();
});

test('ListComponent shallow', () => {
  console.log(mount(List).html());
  console.log(shallowMount(List).html());
});
