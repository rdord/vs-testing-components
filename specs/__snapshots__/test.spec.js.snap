// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`first spec 1`] = `
<div>
  <h1>Hello my value</h1>
</div>
`;
